pylint/reporters/text.py

```diff
@@ -35,8 +35,11 @@
 from pylint.reporters.ureports.text_writer import TextWriter
 
 if TYPE_CHECKING:
+    from pylint.lint import PyLinter
     from pylint.reporters.ureports.nodes import Section
 
+ColorMappingDict = Dict[str, Tuple[Optional[str], Optional[str]]]
```

Comment:
Additionally, the `style` attribute could be `tuple[str, ...]` instead of `str | None`.
Just need to check if we need to preserve some backwards compatibility.